api/src/api/emailCheckRouter.ts

```diff
@@ -0,0 +1,34 @@
+import { UserDataClient } from "@domain/types";
+import type { LogWriterType } from "@libs/logWriter";
+import { Router } from "express";
+import { StatusCodes } from "http-status-codes";
+
+export const emailCheckRouterFactory = (userDataClient: UserDataClient, logWriter: LogWriterType): Router => {
+  const router = Router();
+
+  router.post("/emailCheck", async (req, res) => {
```

Comment:
[pebble] Given we're not actually modifying any information, really just trying to assert whether a value exists or not, I think this should be a `GET`.
```suggestion
  router.get("/emailCheck", async (req, res) => {
```

Reply:
I’m fine with that. The original spec had the email being searched being supplied via request body. I think a GET + query parameter makes more semantic sense for what we’re trying to do here, so updating to that approach

Reply:
@jphechter Spec on the ticket says `POST`, and that was my choice. `GET` requests are often cached, and rather than spend time on doing cache busting and diagnostics around cache busting, it was easier to choose `POST` even though it's not REST-y, since `POST` requests don't get cached by default.

If it was an authenticated endpoint, then I'd say `GET` all the way. Open to the conversation tho.

@hinzed1127 FYI, it's unusual but not non-standard to send a body in a `GET` request. `GET` doesn't _necessarily_ require query parameters. Still, my suggestion would be to stick with a `POST` request.

Reply:
Cool with me. Semantically, I'd lean on `GET` here, but for ease of implementation I don't have a strong feeling. IF we ever do come back and start to build out our API in some way, it is a change I'd like to see happen.

But that's probably a long, long time from now.